src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -196,15 +196,22 @@ public static String formatDate(final long timestamp) {
 	
 	public static final Option<Boolean> enableTimings = new Option<>("enable timings", false)
 			.setter(t -> {
-				if (Skript.classExists("co.aikar.timings.Timings")) { // Check for Paper server
-					if (t)
-						Skript.info("Timings support enabled!");
-					SkriptTimings.setEnabled(t); // Config option will be used
-				} else { // Not running Paper
+				if (!Skript.classExists("co.aikar.timings.Timings")) { // Check for Paper server
 					if (t) // Warn the server admin that timings won't work
 						Skript.warning("Timings cannot be enabled! You are running Bukkit/Spigot, but Paper is required.");
 					SkriptTimings.setEnabled(false); // Just to be sure, deactivate timings support completely
+					return;
 				}
+				if (Skript.isRunningMinecraft(1, 19, 4)) { // check for valid paper version (<1.19.4)
```

Comment:
Would it be better to check for the deprecation annotation?

Reply:
It may, but honestly I just didn't know that was possible.

Reply:
After [discussion](https://discord.com/channels/135877399391764480/828337517471203328/1141856381159362610), decided to keep it as is. Possible danger if, as Pickle says, "other platforms end up using it / keeping it but that seems unlikely"